flexible-classroom/reference/platform-support.md

```diff
@@ -0,0 +1,84 @@
+---
```

Comment:
Suggestion: Could you please add a doc called "Edu Context Client API" before this platform support doc? For the new API doc, you can directly refer to the API page. 

Reason: The background is that Flexible Classroom has Classroom Client API, Edu Context Client API, and Classroom Cloud Service RESTful API. For the relationship and difference between these APIs, you can read the technical architecture diagram in the reference doc. If you put just two API docs in the reference section and put another API doc on the API page, it's can hard for users to find all the API docs they need to use. Besides, I wonder if we can add the Classroom Client API doc to the API page as well.

Reply:
Can you supply the source please.

Reply:
> Can you supply the source please.

You mean the source of Edu Context Client API? We provide it by sending an HTML zip folder to you for rendering the API reference webpage before. Now, you can also check it from this link: https://github.com/AgoraIO/API-Reference/tree/main/flexible-classroom-sdk.

Reply:
Ok, I've added a page with links to the ref.